src/components/v2/Modal/ModalStyles.ts

```diff
@@ -0,0 +1,61 @@
+import { useTheme } from '@mui/material';
+
+const iconCloseSize = '35px';
+
+export const useModalStyles = ({
+  hasTitleComponent,
+  noHorizontalPadding,
+}: {
+  hasTitleComponent: boolean;
+  noHorizontalPadding?: boolean;
+}) => {
+  const theme = useTheme();
+  return {
+    box: {
+      position: 'absolute' as const,
+      top: '50%',
+      left: '50%',
+      transform: 'translate(-50%, -50%)',
+      width: '80%',
+      maxWidth: 544,
+      boxShadow: 24,
+      padding: 0,
+      borderRadius: theme.spacing(3),
+      paddingTop: theme.spacing(3),
+    },
+    titleWrapper: {
+      position: 'relative',
+      paddingLeft: theme.spacing(3),
+      paddingRight: theme.spacing(3),
+      paddingBottom: hasTitleComponent ? theme.spacing(3) : 0,
+      borderBottom: hasTitleComponent ? `1px solid ${theme.palette.secondary.dark}` : 0,
+      marginBottom: hasTitleComponent ? theme.spacing(3) : 0,
+      paddingTop: 0,
+    },
+    titleComponent: {
+      alignSelf: 'center',
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      minHeight: iconCloseSize,
+      paddingLeft: iconCloseSize,
+      paddingRight: iconCloseSize,
+    },
+    closeIcon: {
+      right: theme.spacing(3),
+      top: 0,
+      position: 'absolute',
+      height: iconCloseSize,
+      width: iconCloseSize,
+      minWidth: iconCloseSize,
+      marginLeft: 'auto',
+      padding: 0,
+    },
+    contentWrapper: {
+      padding: theme.spacing(3),
+      paddingLeft: noHorizontalPadding ? 0 : theme.spacing(3),
+      paddingRight: noHorizontalPadding ? 0 : theme.spacing(3),
+      paddingTop: 0,
```

Comment:
```suggestion
      padding: theme.spacing(0, noHorizontalPadding ? 0 : 5, 5, noHorizontalPadding ? 0 : 5),
```

<img width="749" alt="Screenshot 2022-03-25 at 13 06 41" src="https://user-images.githubusercontent.com/44675210/160126322-d63577a2-5c18-4d7f-af94-c0a7ddce6239.png">

Reply:
for me, it's easier to read current padding props